### PR TITLE
PKPDX-11 PKPDX-13

### PR DESCRIPTION
**Pull request by wniestroj**
PKPDX-11:
image

PKPDX-13:
image

### CODE REVIEW

**Comment by martinjrobins** [comment]
Thanks @wniestroj, I'll let @eatyourgreens review this

File: frontend-v2/src/features/data/LoadData.tsx
Comment (eatyourgreens) [inline frontend-v2/src/features/data/LoadData.tsx:193]: If I upload the wrong file by mistake, then `showData` will be true so I won't be able to correct my mistake and upload the correct file. Is that intentional?

Comment (wniestroj) [reply frontend-v2/src/features/data/LoadData.tsx:193]: Yes. According to the mockups, there will be an additional button added next to 'Previous/Next' called 'Cancel' - which will allow users to cancel working on current file and upload a new one. On user cancelation showData prop will be set back to false for the dropdown to appear.

Mockups - https://www.figma.com/design/bxDmCsAM7vIFpe6mBBJ9p2/Pathfinder?node-id=457-82&node-type=canvas&t=lGqrIgh9gTjLUPPY-0 - Desktop - S1 - option1 and the one below showcasing the tooltip

Comment (eatyourgreens) [reply frontend-v2/src/features/data/LoadData.tsx:193]: Cool! In that case this looks good. My only comment is that it's forbidden to embed one clickable element inside another clickable element in HTML, so the button inside the clickable div might be invalid.

**Review by eatyourgreens — APPROVED** [review]

File: frontend-v2/src/features/data/LoadData.tsx
Comment (wniestroj) [reply frontend-v2/src/features/data/LoadData.tsx:193]: That's a valid point, will update it on Monday before merging.

**Pull request merged**
